setup.cfg

```diff
@@ -1,4 +1,7 @@
 [tool:pytest]
 python_files=test*.py
 addopts=--tb=short
+[flake8]
+max-line-length = 180
```

Comment:
I'd suggest 120 will good in case of using type hinting etc, but 180 would be very long and especially for code review that would require horizontal scrolling.

Reply:
I can do that, but there are already many lines longer than that in the code. Too many long lines errors in the report was obscuring other more important errors, that's why I added this to the `setup.cfg`

But I can remove this commit from the PR entirely and solve all flake issues in separate PR and add it to the Travis testing, if you are interested.

Reply:
Hmm, that's unfortunate.
In that case, 180 is OK, the line count can be dealt with in a separate pull request, handling those long lines would pollute the pull request.

Let's carry on then.